### DISCUSSION TITLE
BattleBoxes Multiplayer Release 1.3.0

### DISCUSSION THREAD

**Discussion by spsquared** [spsquared/BattleBoxes_Multiplayer#132] [Releases]
| 1.2.1 | <ul><li>Added map name display at round start</li><li>Randomly spawned lootboxes</li><li>Added textures to lootbox</li><li>Added "slowness" effect to lootbox rng</li><li>Added achievements for loot boxes</li><li>Changed banner outline from dark grey to black</li><li><a href="https://github.com/definitely-nobody-is-here/BattleBoxes_Multiplayer/issues/102" target="_blank">Fixed win screen flashing when zooming in and out (Issue #102)</a></li><li>Improved page loading</li><li>Increased spam cooldown</li></ul> |
| 1.2.2 | <ul><li>Added secondary attacks (found in loot boxes)</li><li>Added golden, super, laser, and smart variants of bullets</li><li>Added "The Golden Bullet" achievement</li><li>Added "yeet" attack</li><li><a href="https://github.com/definitely-nobody-is-here/BattleBoxes_Multiplayer/issues/124" target="_blank">Fixed win screens on players who are not in game (Issue #124)</a></li><li><a href="https://github.com/definitely-nobody-is-here/BattleBoxes_Multiplayer/issues/120" target="_blank">Fixed instant win caused by un-reset scoreboards (Issue #120)</a></li><li><a href="https://github.com/definitely-nobody-is-here/BattleBoxes_Multiplayer/issues/126" target="_blank">Hid play again button on game join (Issue #126)</a></li><li><a href="https://github.com/definitely-nobody-is-here/BattleBoxes_Multiplayer/issues/125" target="_blank">Fixed two players winning (Issue #125)</a></li><li><a href="https://github.com/definitely-nobody-is-here/BattleBoxes_Multiplayer/issues/121" target="_blank">Fixed broken divs when in win screen (Issue #121)</a></li><li>Note: Pathfind bullets do not work</li></ul> |
| 1.3.0 | <ul><li>Added "Rain" map by The-God-Coder</li><li>Added "The Labyrinth" map</li><li><a href="https://github.com/definitely-nobody-is-here/BattleBoxes_Multiplayer/issues/129" target="_blank">Cleared secondary attack each round (Issue #129)</a></li><li>Improved buff timeouts</li></ul> |

<hr /><em>This discussion was created from the release <a href='https://github.com/definitely-nobody-is-here/BattleBoxes_Multiplayer/releases/tag/v1.3.0'>BattleBoxes Multiplayer Release 1.3.0</a>.</em>

**Comment by SampleProvider** [comment]
haha rain go brrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrr

**Comment by SampleProvider** [comment]
s

**Comment by SampleProvider** [comment]
asddddddd
